wiki/Why-This-Library.md

```diff
@@ -0,0 +1,20 @@
+### Why This Library?
+There are many reasons to be excited about Workflow:
+- Build flexibility into your applications so that you can easily make changes when you need to.
+- Compose new `Workflow`s to quickly create a new journey through your app.
+- Stop components in your app from knowing more than they should, no more complicated routers or views that have to know what comes next.
+- Create flows that give your users a better experience, web frameworks can easily sign somebody in, or sign them up, then continue into a flow, so why not be able to easily do the same in Swift?
+- Power deep linking experiences that can get your users right where you want them with minimal overhead.
+
+Read on to learn about our philosophy of development. We believe these ideals take us from "just another library" too a tool you won't want to do without.
+
+### Developer Experience over Problem Solving
+There have been many attempts by others to create libraries that decouple the concept of a flow that can move forward and backwards, but they all took very different approaches. What separates us, and makes this library worth using, is our developer-centric experience. We sat down with a group of developers and asked the question "if something existed today that did what you wanted, what would that thing look like?" then we spent a lot of effort making that happen.
+
+Additionally, this is a concerted effort from WWT. We do a lot of software development for large companies and this problem space is something we've regularly encountered. The concept of a `Workflow` has been toyed with by several projects. This library has been used to power some really great experiences already, and we intend to continue using it to do even bigger and better things in the future.
+
+### Flexibility over Assumptions
+Not everyone develops software the same way, and it would be a sad day for the industry if we thought we had all the answers. We believe strongly in a culture of innovation and we want consumers of our library to be able to make choices they feel are correct. `Workflow`s are not an architectural pattern, they're an abstraction you can choose to use. Much like the standard libraries that come with programming languages the idea here is to give you tools that can be really useful and let you choose the best way to use them.
```

Comment:
This falls under one of the previous 2 Workflow comments. Decide which one and fix it.

Reply:
I decided concept, and removed formatting.